RetailCoder.VBE/Common/RubberduckHooks.cs

```diff
@@ -40,21 +32,19 @@ public void HookHotkeys()
             var config = _config.LoadConfiguration();
             var settings = config.UserSettings.HotkeySettings;
 
-            foreach (var hotkey in settings.Settings.Where(hotkey => hotkey.IsEnabled))
+            foreach (var hotkeySetting in settings.Settings.Where(hotkey => hotkey.IsEnabled))
```

Comment:
The name `hotkey` used in the LINQ expression threw me for a loop - I think it should be renamed `hotkeySetting`.

Reply:
Done